packages/components/src/__future__/ExampleComponent/docs/ExampleComponent.stories.tsx

```diff
@@ -0,0 +1,14 @@
+import React from "react"
```

Comment:
I just realised, this structure actually doesn't match the other existing packages either. Those are something like:

```
package-name-here
├── docs
|   ├── ExampleComponent.stories.tsx
├── src
│   ├── ExampleComponent
|   │   ├── ExampleComponent.tsx
|   |   ├── ExampleComponent.spec.tsx
|   │   ├── ExampleComponent.module.scss
│   └──index.ts
```

Does this mess everything up? I do admit I don't like it and I much prefer the docs being within the component at the very least (I think I read your initial proposal wrong and I am less against it now).

Reply:
Ah my bad, I should've framed the proposal from the perspective of the AIO. But yeah, the docs dir sits with the component.

Reply:
Shippit. I do like the `__docs__` syntax too to surface them. Not used to `__tests__` but I could live with it. Although the subcomponent structure still makes me worry about having a million nested folders, but this is for the other discussion.